README.md

```diff
@@ -118,6 +117,21 @@ rdd.toHBaseTable("mytable")
 
 You are done. HBase now contains *100* rows in table *mytable*, each row containing two values for columns *mycf:column1* and *mycf:column2*.
 
+### With Timestamp:
+
+If the last column match **timestamp** will be set as timestamp on the Hbase record, the others will be assigned to columns.
```

Comment:
Better: "If the last column name matches **timestamp**". Is "timestamp" a reserved word or a user are allowed to call a regular column "timestamp"?

Reply:
@nicolaferraro  the best approach that I could figure out was to include the reserved word and the last column matches `timestamp`, I didn't like so much but seems to work. 
What do you think? 

I will fixed the text also

Reply:
This is useful in the cases where you want to set a custom timestamp per row and not a equal timestamp for all elements of the RDD. If timestamp is not a reserved word, I'd add a flag in the DSL to enable/disable this behaviour.

Something like:

```
rdd.toHBaseTable("mytable")
    .toColumns("column1", "column2", "timestamp")
    .inColumnFamily("mycf")
    .usingTimestampColumn() // <--
    .save()
```

What do you think?

Reply:
@nicolaferraro  maybe It's better to create a flag to set up the timestamp to the record, this wouldn't break current applications and will have less impact on the current API. 

``` 
rdd.toHBaseTable("mytable")
    .toColumns("column1", "column2", "timestamp")
    .inColumnFamily("mycf")
    .usingTimestampRecord() // <--
    .save()
``` 

WDYT?

Reply:
Yes, sounds good!